Description bug fix and test file deleted

kKaskak opened the pull request:
Description not showing up in the movie/serie details page not showing up fixed

kKaskak committed:
Fixed description on details page  not showing up

kKaskak committed:
Test file deleted

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/Stremio/stremio-web?pullRequest=461) <br/>All committers have signed the CLA.

kKaskak committed:
compact check for the desc removed

kKaskak committed:
Delete .prettierignore

tymmesyde reviewed (commented):
Now that i think about it, we could make it look like the desktop version: under the tags with the summary title on the Detail page
But keep it like this for the compact version like on the Discover page

kKaskak committed:
Update MetaPreview.js

kKaskak committed:
Conditional rendering description

Two different cases: Rendering the description on the discover page before the genres and rendering the desc after the genres on the Details page

kKaskak committed:
Removed spacing in the file

kKaskak committed:
Added "Summary" label

kKaskak committed:
Update styles.less

kKaskak committed:
Added translation to the label

Fixed spacing

kKaskak committed:
indetation fix

kKaskak committed:
indetation fix 2

kKaskak committed:
indentation fix 3

kKaskak committed:
bracket indentation fix

kKaskak committed:
string fix to one quote

kKaskak committed:
space fix indentation
